### PR TITLE
Update org.eclipse.osgi version from 3.15.0.v20190725-1512 to 3.15.0.v20190808-1644

### PR DESCRIPTION
**Pull request by dazavala**
Update org.eclipse.osgi version from 3.15.0.v20190725-1512 to 3.15.0.v20190808-1644, and verify open-liberty supports embeddeding an OSGi framework instance within a web application.

### CODE REVIEW

**Comment by dazavala** [comment]
#build

File: dev/com.ibm.ws.kernel.boot_fat/test-applications/checkJvmAppClasspath/resources/META-INF/permissions.xml [hunk 1]
```diff
@@ -17,7 +17,7 @@
     version="7">
 	
    <permission>
-   		<class-name>java.lang.RuntimePermission</class-name>
+   		<class-name>java.security.AllPermission</class-name>
```
Comment (tjwatson) [inline dev/com.ibm.ws.kernel.boot_fat/test-applications/checkJvmAppClasspath/resources/META-INF/permissions.xml]: Why did the permissions need updated for this checkJvmAppClasspath application?

Comment (dazavala) [reply dev/com.ibm.ws.kernel.boot_fat/test-applications/checkJvmAppClasspath/resources/META-INF/permissions.xml]: The test failed consistently in sandbox runs due a lacking permission. I widened the application's permission set with the concern that we ensure the problem does not surface in other FATs during full regression runs.  We'll know soon.

**Comment by dazavala** [comment]
#build

**Comment by dazavala** [comment]
#build

**Comment by dazavala** [comment]
Added changes to resolve the feature verification error.

#build

**Pull request merged**